content/kotsadm/gitops/multi-app-workflows.md

```diff
@@ -6,46 +6,42 @@ weight: 2
 draft: false
 ---
 
-While the Admin Console is initially configured to receive updates, show the changes and deploy them to the cluster, this process can be changed and converted to use a GitOps workflow instead. 
+While the Admin Console is initially configured to receive updates, show the changes and deploy them to the cluster, this process can be changed and converted to use a GitOps workflow instead.
 When using a GitOps workflow, changes from the Admin Console (config changes, upstream updates, license updates) will be pushed to a private Git repository, where an existing CI/CD process can execute to deliver the manifests to the cluster.
 
-To begin migrating to a GitOps deployment workflow, click the GitOps link at the top of the Admin Console, then click on "Get started".
+To begin migrating to a GitOps deployment workflow:
 
-![GitOps](/images/gitops.png)
+1. Click the GitOps link at the top of the Admin Console, then click on "Get started".
 
-Continuing, choose the Git provider and hostname (if applicable), then click on "Finish GitOps setup".
+    The GitOps Provider page opens.
```

Comment:
Delete- I added this line to the first step. I'm indifferent if you feel strongly about keeping it on a separate line, but I normally do in the same step with a line break.